.travis.yml

```diff
@@ -11,19 +11,20 @@ matrix:
   allow_failures:
     - env: TOXENV=isort
 before_cache:
-  - rm -f $HOME/.cache/pip/log/debug.log
+  - pyclean $HOME/virtualenv/python2.7.9/lib
 cache:
   directories:
     - pootle/static/js/node_modules
     - pootle/assets
-    - $HOME/.cache/pip
+    - $HOME/virtualenv/python2.7.9/lib
+    - $HOME/virtualenv/python2.7/bin
```

Comment:
Really?  This venv comes with the python Travis build so we'll be reverting any fixes that they make.  Is that a good thing?